native/acter/api.rsh

```diff
@@ -687,6 +687,13 @@ object Room {
     /// get the room profile that contains avatar and display name
     fn space_relations() -> Future<Result<SpaceRelations>>;
 
+    /// add the following as a parent room
+    /// it can have multiple parents
+    fn add_parent_room(room_id: string, canonical: bool) -> Future<Result<string>>;
```

Comment:
what is that resulting string? Should probably also be added to the documentation...

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1089/commits/ce888c1f02eadbd4cc2f79e7211e0cd0c004331b